src/blocks/media-block/ImageBlock.test.tsx

```
@@ -0,0 +1,63 @@
+    const image = screen.getByTestId('mock-image');
+    fireEvent.click(image);
+    expect(handleClick).toHaveBeenCalledTimes(1);
+  });
+
+  it('passes through additional props', () => {
+    render(<ImageBlock {...defaultProps} data-testid="custom-test-id" />);
+    expect(screen.getByTestId('custom-test-id')).toBeInTheDocument();
+  });
+});
+
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)